internal/pkg/resourceconfig/resourceconfig.go

```diff
@@ -134,29 +136,70 @@ func (p *RawConfigFileProvider) IsSupported(path string) bool {
 }
 
 // GetConfig returns the resource configs
-func (p *RawConfigFileProvider) GetConfig(path string) ([]*unstructured.Unstructured, error) {
+// from a directory or a file containing raw Kubernetes resource configurations
+func (p *RawConfigFileProvider) GetConfig(root string) ([]*unstructured.Unstructured, error) {
 	var values clik8s.ResourceConfigs
 
-	b, err := ioutil.ReadFile(path)
-	if err != nil {
-		return nil, err
-	}
-	objs := strings.Split(string(b), "---")
-	for _, o := range objs {
-		body := map[string]interface{}{}
+	err := filepath.Walk(root, func(path string, fi os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
 
-		if err := yaml.Unmarshal([]byte(o), &body); err != nil {
-			return nil, err
+		if fi.IsDir() {
+			return nil
 		}
-		values = append(values, &unstructured.Unstructured{Object: body})
-	}
 
+		b, err := ioutil.ReadFile(path)
+
+		if err != nil {
+			return err
+		}
+		objs := strings.Split(string(b), "---")
```

Comment:
I believe yaml library would handle this and return a list of unstructured.
Could you confirm

Reply:
I tried and it couldn't load to a list of unstructured.